R/score.R

```diff
@@ -251,6 +251,14 @@ calculate_scores_rowwise <- function(reviews, submissions) {
     ) %>%
     dplyr::inner_join(submissions, by = c("submission", "step", "form_data_id")) %>%
     dplyr::mutate(step2 = .data$step) %>%
+    dplyr::mutate(
+      section_flag = dplyr::case_when(
+        .data$section %in% c("binding", "efficacy", "in_vivo_data", "pk_in_vivo", 
+                             "acute_dosing", "chronic_dosing", "teratogenicity",
+                             "toxicology") ~ 0,
+        TRUE ~ 1
```

Comment:
is `TRUE ~1` equivalent to `.default = 1` as described in the [case_when](https://dplyr.tidyverse.org/reference/case_when.html) docs?

Reply:
Yes it is.... the only reason i used the TRUE ~ 1 syntax is that is being used quite often elsewhere in the app already, so I just kept it consistent.